src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/Symmetric/SymmetricOneShotBase.cs

```diff
@@ -469,6 +470,7 @@ public void DecryptOneShot_Cbc_InvalidPadding_DoesNotContainPlaintext(PaddingMod
         [InlineData(PaddingMode.PKCS7, 2048)]
         [InlineData(PaddingMode.ANSIX923, 2048)]
         [InlineData(PaddingMode.ISO10126, 2048)]
+        [SkipOnPlatform(TestPlatforms.Browser, "CipherMode.ECB is not supported on Browser")]
```

Comment:
I see a bunch of suppressions about ECB and CFB not being supported, but no test that verifies the experience.  Please add such a test.

Reply:
I think those are tested in `AesTests.Browser`, no? https://github.com/dotnet/runtime/blob/e087e9520dc4d5c178e5cdafab2ea5aa487b2061/src/libraries/System.Security.Cryptography/tests/AesTests.Browser.cs

Reply:
I see that it was just at the other end of the PR.